src/Squidex/app/shared/services/schemas.service.ts

```diff
@@ -473,6 +491,17 @@ export class DateTimeFieldPropertiesDto extends FieldPropertiesDto {
 
         return validators;
     }
+
+    public getDefaultValue(): any {
+        if (this.calculatedDefaultValue != null) {
+            let now = new Date();
+            if (this.calculatedDefaultValue === 'Today') {
+                now = new Date(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate());
```

Comment:
The implementation is different from the server implementation where we would only return dates for Today (no Time). Would be good to create some tests for it (and use DateTime class).